uitools/examples/cpp_quick/src/demos/TimeSliderDemoModel.cpp

```diff
@@ -0,0 +1,14 @@
+#include "TimeSliderDemoModel.h"
+
+#include "ArcGISMapImageLayer.h"
+#include "Map.h"
+#include "Scene.h"
+
+TimeSliderDemoModel::TimeSliderDemoModel(QObject* parent) :
+  BaseDemoModel(parent)
+{
+}
+
+TimeSliderDemoModel::~TimeSliderDemoModel()
+{
+}
```

Comment:
a lot of files don't have a new line at the end. That causes some IDEs and things to grumble...

Reply:
Are you sure about that one? I'm not seeing the problem in git, and they have the newlines in QtCreator.

Reply:
maybe github hides the newlines now 🤷‍♂️